src/authentication/SimpleAuthentication/IAuthProviderCallback.ts

```diff
@@ -5,11 +5,14 @@
  * -------------------------------------------------------------------------------------------
  */
 
-import { AuthProviderCallback } from "./IAuthProviderCallback";
 
-/**
- * @interface
- * Signature that holds authProvider
- * @callback - The anonymous callback function which takes a single param
- */
-export type AuthProvider = (done: AuthProviderCallback) => void;
+export interface AuthProviderCallback {
```

Comment:
What's the value of this interface since we already have access token provider in kiota? (or will have soon at least)